src/NLog.Extensions.Logging/Logging/NLogProviderOptions.cs

```diff
@@ -68,6 +68,11 @@ public class NLogProviderOptions
         /// <remarks>This option affects the building of service configuration, so assigning it from appsettings.json has no effect (loaded after).</remarks>
         public bool ReplaceLoggerFactory { get; set; }
 
+        /// <summary>
+        /// Checks the Host Configuration for the specified section-name, and tries to load NLog-LoggingConfiguration after creation of NLogLoggerProvider
+        /// </summary>
+        public string LoadConfigurationFromSection { get; set; }
```

Comment:
I expected a boolean here with this name. Maybe something like ConfigurationSectionName?

Reply:
It will only load configuration when having specified a SectionName. The reason for having it as a string is because of this guy: https://github.com/NLog/NLog.Extensions.Logging/pull/488

Reply:
With NLog 5.0 then the default-value can be changed to "NLog"

Reply:
Could also be `LoadLoggingConfigurationFromSectionName`

Reply:
Why do we need the "load" in the name?

Reply:
> The reason for having it as a string is because of this guy: #488

I mean, I expected a boolean because of the name. String is OK, if it has a clear name.

Reply:
Will change it to `LoggingConfigurationSectionName`

Reply:
Fixed